spec/features/team/show_team_spec.rb

```diff
@@ -21,4 +21,22 @@
       expect(page).to have_text another_user.first_name
     end
   end
+
+  describe 'display consecutive numbering of members' do
+    before(:each) do
+      sign_in @user
+    end
+
+    it 'should not show an id column for members' do
+      @team.members << @user
+      visit team_path @team
+      expect(page).to_not have_text "Id"
```

Comment:
You maybe want to test the expected test with a translation thing and no hardcoded string.